client/verta/tests/unit_tests/strategies.py

```diff
@@ -194,11 +196,13 @@ def _build_scan_detail(draw) -> Dict[str, Any]:
 
 
 @st.composite
-def build_scan_dict(draw) -> Dict[str, Any]:
-    """Generate a Verta build scan, as returned by /api/v1/deployment/builds/{build_id}/scan."""
+def build_scan_dict(draw, external_scan: bool = False) -> Dict[str, Any]:
```

Comment:
I know this is a bit of extra work, but I'd recommend

```suggestion
def build_scan_dict(draw, external_scan: Optional[bool] = None) -> Dict[str, Any]:
```

What if we wanted to force `external_scan=False`?

Reply:
Or we can file a follow-up ticket and get this PR in first.

Reply:
@ewagner-verta I might have been unclear on this feedback. This one line change alone doesn't address my core comment of

> What if we wanted to force external_scan=False?

We also need to change the body of this function, because

```
"scan_external": True if external_scan else draw(st.booleans()),
```

doesn't facilitate forcing `False` at all. We should've also changed the `build_dict` strategy as well